### PR TITLE
feat(logo): adds D logo w/ darkmode support

### PR DESCRIPTION
**Pull request by Pardoz**
Hello!
This is the proposal for Dats logo in two flavours 🍦 , the standard one and with the alternate version for dark-mode, following the [supported url syntax](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) :octocat: .

What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,24 +18,25 @@ This client aims to have a simple [statsd](https://github.com/statsd/statsd) yet
...
-        -   [`Client.connect()`](#clientconnect)
-        -   [`Client.counter(string[, value, sampling])`](#clientcounterstring-value-sampling)
-        -   [`Client.timing(string, value[, sampling])`](#clienttimingstring-value-sampling)
-        -   [`Client.gauge(string, value)`](#clientgaugestring-value)
-        -   [`Client.set(string, value)`](#clientsetstring-value)
--   [Benchmarks](#benchmarks)
--   [Powered Apps](#powered-apps)
--   [Support & Contribute](#support--contribute)
--   [License](#license)
+- [Table of Content](#table-of-content)
```
Comment (JellyBellyDev) [inline README.md:21]: please, remove this line! ;)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<p align="center"><img src="./logo.png" alt="logo" width="200px" /></p>
+<p align="center"><img src="./logo.svg?sanitize=true#gh-light-mode-only" alt="logo" width="200px" /><img src="./logo-on-dark.svg?sanitize=true#gh-dark-mode-only" alt="logo" width="200px" /></p>
```
Comment (JellyBellyDev) [inline README.md:1]: Very 🆒 !!!

**Comment by antoniomuso** [comment]
Very Cool, Good job! Thank you!
LGTM!

**Comment by simonecorsi** [comment]
Cool! LGTM!

File: README.md (hunk 1 above)
Comment (Pardoz) [reply README.md]: done 😄

**Review by JellyBellyDev — APPROVED** [review]

**Review by antoniomuso — APPROVED** [review]

**Pull request merged**